python/tank/pipelineconfig_factory.py

```diff
@@ -214,11 +215,12 @@ def _from_path(path, force_reread_shotgun_cache):
     # create a pipeline configuration
     return _validate_and_create_pipeline_configuration(
         associated_sg_pipeline_configs,
+        sg_data["projects"].values(),
         source=path
     )
 
 
-def _validate_and_create_pipeline_configuration(associated_pipeline_configs, source):
+def _validate_and_create_pipeline_configuration(associated_pipeline_configs, projects, source):
```

Comment:
pls add docstring

Reply:
Actually it's not even needed anymore! Taking it out.